Makefile

```diff
@@ -1,5 +1,9 @@
 up-node:
 	docker-compose -f docker-compose.yml -f docker-compose.node.yml up --build
 
+up-java:
+	mvn -f java-backend/ package
```

Comment:
I'm not able to execute this on my local machine, maybe we can include this into the container?
<img width="304" alt="image" src="https://user-images.githubusercontent.com/56513947/228995840-acc4dd85-0779-4e2e-ac02-c7725b306859.png">

Reply:
you need to install[ mvn](https://maven.apache.org/download.cgi) add it to the path, its similar to npm 👍🏾